db/init.sql

```diff
@@ -0,0 +1,28 @@
+DROP TABLE IF EXISTS Reservations;
+DROP TABLE IF EXISTS Events;
+DROP TABLE IF EXISTS Users;
+
+CREATE TABLE Users(
+	id SERIAL PRIMARY KEY,
+	email TEXT,
+	username TEXT,
+	password TEXT,
+	zip TEXT,
+	profile_pic BYTEA,
+	favorite_tags TEXT
```

Comment:
We got rid of this the previous commit, best to remove this to avoid merge conflicts